server/portal/apps/datafiles/views.py

```diff
@@ -29,7 +29,6 @@ class SystemListingView(BaseApiView):
     def get(self, request):
         portal_systems = settings.PORTAL_DATAFILES_STORAGE_SYSTEMS
         local_systems = settings.PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEMS
-
         user_systems = get_user_storage_systems(request.user.username, local_systems)
         # compare available storage systems to the systems a user can access
         response = {'system_list': []}
```

Comment:
Something got messed up with the default system changes here after your merged with master.
Something in this block of code is causing issues: server/portal/apps/datafiles/views.py
```
    def get(self, request):
        portal_systems = settings.PORTAL_DATAFILES_STORAGE_SYSTEMS
        local_systems = settings.PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEMS
        user_systems = get_user_storage_systems(request.user.username, local_systems)
        # compare available storage systems to the systems a user can access
        response = {'system_list': []}
        for system_name, details in user_systems.items():
            response['system_list'].append(
                {
                    'name': details['name'],
                    'system':  UserSystemsManager(request.user, system_name=system_name).get_system_id(),
                    'scheme': 'private',
                    'api': 'tapis',
                    'icon': details['icon']
                }
            )
        ######### This Part:
        # This does not line up with what your committed earlier on.
        response['system_list'] += portal_systems
        default_system = user_systems[settings.PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEM_DEFAULT]
        response['default_host'] = default_system['host']

        for system in response['system_list']:
            try:
                if system['api'] == 'tapis' and 'system' in system:
                    system['definition'] = StorageSystem(
                        request.user.agave_oauth.client, id=system['system']
                    )
            except Exception:
                logger.exception("Could not retrieve definition for {}".format(system['system']))

        return JsonResponse(response, encoder=BaseAgaveSystemSerializer)